Pegasus/include/Math.hpp

```diff
@@ -1019,6 +1022,10 @@ class JacobiEigenvalue
     void Calculate();
 };
 
+glm::dvec3 CalculateCentroid(std::initializer_list<glm::dvec3> vectors);
```

Comment:
Why you using this explicit? Write glm::dvec3 const& vectors here and provide { } to CalculateCentroid like 
CalculateCentroid({{1..1}, {2..2}}) like this